src/components/inputs/async-creatable-select-input/async-creatable-select-input.js

```diff
@@ -47,6 +47,8 @@ const AsyncCreatableSelectInput = props => {
               hasError: props.hasError,
               showOptionGroupDivider: props.showOptionGroupDivider,
               menuPortalZIndex: props.menuPortalZIndex,
+              isDisabled: props.isDisabled,
+              isReadOnly: props.isReadOnly,
```

Comment:
We're now also passing the `isDisabled` prop because we will use it for the styles instead of the state, to differentiate between `isDisabled` and `isReadOnly`.